app/src/main/java/com/eveningoutpost/dexdrip/Models/Calibration.java

```diff
@@ -906,6 +908,11 @@ public static void adjustRecentBgReadings(int adjustCount) {
             try {
                 final Calibration latestCalibration = Calibration.lastValid();
                 int i = 0;
+                boolean syncAdjustedReadings = false;
+                boolean uploadAdjusted = Pref.getBoolean("upload_modified_bgreadings", false);
+                if (uploadAdjusted) {
+                    Log.v(TAG, "Adjusted readings will be uploaded to NS");
```

Comment:
I think the modified BG values will not just be uploaded to NS. They will be synced with any active sync service.

Reply:
More precisely, BG values will be synced to the following services, if active: https://github.com/NightscoutFoundation/xDrip/blob/38fee03be56da6ce9cbe90e08dbe8e4b651c6edc/app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/UploaderQueue.java#L87-L91